src/styles/blocks/container.scss

```
@@ -0,0 +1,40 @@
+.container {
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(300px, 1fr));
+  grid-auto-rows: 300px;
+
+  @media(min-width: 900px) {
+    grid-template-columns: auto auto auto;
+  }
+
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 100px;
+  color: white;
+
+  line-height: 300px;
+  text-align: center;
+
+  &__block {
+    width: 100%;
+    height: 100%;
```

Насчет внутреннего центрирования
```suggestion
    display: flex;
    justify-content: center;
    align-items: center;
    flex-basis: 300px;
```